src/line-chart/line-series.component.ts

```diff
@@ -42,6 +42,15 @@ import { sortLinear, sortByTime, sortByDomain } from '../utils/sort';
         [class.active]="isActive(data)"
         [class.inactive]="isInactive(data)"
       />
+     <svg:g ngx-charts-line
```

Comment:
We should use an area component here, not line. Let's change this to `ngx-charts-area` and remove the customization of the line component.